database/influx/influx.js

```diff
@@ -0,0 +1,61 @@
+const Influx = require("influx");
+const logger = require("./../../lib/log")(__filename);
+require("dotenv").config();
+
+const influxModule = {};
+let client;
+const HOST = process.env.INF_HOST||"localhost";
+const PORT = process.env.INF_PORT||8086;
+influxModule.startInflux = () => {
+  client = new Influx.InfluxDB({
+    host: HOST,
+    port: PORT,
+  });
+  logger.info("Sucessfully started InfluxDB");
+};
+
+influxModule.createAccount = async (account) => {
+  const { username, dbPassword } = account;
+  if (!username || !dbPassword) return;
+  try {
+    await client.createUser(username, dbPassword);
+    await client.createDatabase(username);
+    await client.grantPrivilege(username, "WRITE", username);
+    logger.info(
+      `Sucessfully created new user and database with ${username} name`
+    );
+  } catch (err) {
+    logger.error(err);
+    throw new Error(`failed to create new influx user with ${username} name`);
+  }
+};
+influxModule.deleteAccount = async (account) => {
```

Comment:
This function only uses `username`, so why is the entire `account` object needed?